### PR TITLE
chore: Expose methods from Flow crud classes to browser callables

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

**Comment by Artur-** [comment]
This is related to https://github.com/vaadin/flow/pull/20743

File: packages/java/parser-jvm-plugin-backbone/src/main/java/com/vaadin/hilla/parser/plugins/backbone/EndpointExposedPlugin.java [hunk 1]
```diff
@@ -95,12 +95,20 @@ public NodeDependencies scan(@NonNull NodeDependencies nodeDependencies) {
...
                 .map(AnnotationInfoModel::getName)
-                .anyMatch(endpointExposedAnnotationName::equals);
+                .anyMatch(endpointExposedAnnotationName::equals)
+                || alwaysExpose(classInfo);
         var classInfoNode = exposed ? EndpointExposedNode.of(classInfo)
                 : EndpointNonExposedNode.of(classInfo);
         return classInfoNode;
     }
 
+    private boolean alwaysExpose(ClassInfoModel classInfo) {
```
Comment (cromoteca) [inline packages/java/parser-jvm-plugin-backbone/src/main/java/com/vaadin/hilla/parser/plugins/backbone/EndpointExposedPlugin.java:105]: Could this be treated in a way similar to https://github.com/vaadin/hilla/blob/f4a02726caa01ed7e318ff2d9e2621eaa6347550/packages/java/parser-jvm-core/src/main/java/com/vaadin/hilla/parser/core/Parser.java#L47-L48, i.e. define a list of internal exposed classes there?
I can do the change if you agree.

**Review by cromoteca — CHANGES_REQUESTED** [review]

File: packages/java/parser-jvm-plugin-backbone/src/main/java/com/vaadin/hilla/parser/plugins/backbone/EndpointExposedPlugin.java (hunk 1 above)
Comment (Artur-) [reply packages/java/parser-jvm-plugin-backbone/src/main/java/com/vaadin/hilla/parser/plugins/backbone/EndpointExposedPlugin.java:105]: Sure, if that is a better place to define it

Comment (cromoteca) [reply packages/java/parser-jvm-plugin-backbone/src/main/java/com/vaadin/hilla/parser/plugins/backbone/EndpointExposedPlugin.java:105]: They seem to have something in common to me, but I'm not sure about the place, actually. Let's keep things as they are in your PR.

**Review by cromoteca — DISMISSED** [review]

**Review by cromoteca — APPROVED** [review]

**Pull request merged**